app/(onboarding)/onboarding/new-user/step-3/page.tsx

```diff
@@ -1,21 +1,35 @@
 import { Metadata } from "next";
-import dynamic from 'next/dynamic'
+import dynamic from 'next/dynamic';
 import { HydrationBoundary, QueryClient, dehydrate } from "@tanstack/react-query";
 import { createServerAxiosInstance } from "@/api/server-constructor";
 import { generateBaseQueryKeyFromRoute, routes } from "@/api/routes";
 
-const StepThreeOnboarding = dynamic(() => import('@/app/(public-pages)/_components/onboarding/StepThree'), { ssr: false })
+const StepThreeOnboarding = dynamic(() => import('@/app/(public-pages)/_components/onboarding/StepThree'), { ssr: false });
 
 async function getInitialData() {
   const queryClient = new QueryClient();
   const route = routes.onboarding.getOnboardingPaymentAmount;
+
+  // Validate route
+  if (!route) {
+    throw new Error("Route for onboarding payment amount is not defined.");
+  }
+  console.log("Route URL:", route);
+
```

Comment:
please take this off the try catch is already handling error cases. keeping these lines here would bloat our page